packages/connection/src/class-manager.php

```diff
@@ -861,6 +861,7 @@ public function register( $api_endpoint = 'register' ) {
 				'site_created'    => $this->get_assumed_site_creation_date(),
 				'jetpack_version' => Constants::get_constant( 'JETPACK__VERSION' ),
 				'ABSPATH'         => Constants::get_constant( 'ABSPATH' ),
+				'email'           => wp_get_current_user()->user_email,
```

Comment:
Nitpick - I'd prefer if we were more specific with the naming here. For example, `current_user_email`, `tos_accepted_email`, etc. I could imagine a case where we also want to send the administrator email address as well. Being more explicit will make it easier to distinguish on WP.com.

Reply:
Thanks, I was on the fence. Going with `current_user_email`

Reply:
done, and updated the phab revision